packages/jest-validate/src/__tests__/__snapshots__/validate.test.js.snap

```diff
@@ -135,4 +135,4 @@ exports[`works with custom warnings 1`] = `
 <yellow>  This is probably a typing mistake. Fixing it will remove this message.</>
 <yellow></>
 <yellow>My custom comment</>"
-`;
+`;
```

Comment:
what's up with all these trailing newline changes?

Reply:
I changed it back. My mistake.